code/__DEFINES/dcs/signals/signals_mob/signals_human.dm

```diff
@@ -6,17 +6,13 @@
 #define COMSIG_HUMAN_DISARM_HIT	"human_disarm_hit"	//! Hit by successful disarm attack (mob/living/carbon/human/attacker,zone_targeted)
 #define COMSIG_HUMAN_ATTACKED "carbon_attacked"					//hit by something that checks shields.
 
+///From mob/living/carbon/human/suicide()
+#define COMSIG_HUMAN_SUICIDE_ACT "human_suicide_act"
+
 //Heretics stuff
-#define COMSIG_HERETIC_MASK_ACT "void_mask_act"
-/// From /obj/item/melee/touch_attack/mansus_fist/on_mob_hit : (mob/living/source, mob/living/target)
-#define COMSIG_HERETIC_MANSUS_GRASP_ATTACK "mansus_grasp_attack"
-	/// Default behavior is to use a charge, so return this to blocks the mansus fist from being consumed after use.
-	#define COMPONENT_BLOCK_CHARGE_USE (1<<0)
+/// Default behavior is to use a charge, so return this to blocks the mansus fist from being consumed after use.
+#define COMPONENT_BLOCK_CHARGE_USE (1<<0)
```

Comment:
This is a response from a signal and should be indented. It seems like you moved the actual define that this belongs to somewhere else though, so this should be moved with it. Unless you already moved it, in which case just delete this.